lib/livebook/intellisense.ex

```diff
@@ -506,17 +506,36 @@ defmodule Livebook.Intellisense do
     """
   end
 
-  defp format_hexdocs_link(module, hash \\ "") do
-    hash = if hash == "", do: "", else: "#" <> hash
+  defp format_docs_link(module, function \\ nil, arity \\ nil) do
+    module_string = Atom.to_string(module)
+    first_char = String.at(module_string, 0)
+    is_elixir? = first_char == String.upcase(first_char)
+
+    if is_elixir? do
+      format_hexdocs_link(module, function, arity)
+    else
+      format_erlangdocs_link(module_string, function, arity)
```

Comment:
We can have Erlang docs as a package that also have links on hex.pm, such as hexdocs.pm/telemetry. Instead, I think you want to check if the application belongs to OTP. So in line 524, you want to do something like this:

```
app_path = :code.lib_dir(app)
List.starts_with?(app_path, :code.lib_dir)
```

If it is true, then it is an Erlang/OTP app.